soups/projects/futurelearn-mvp.snip.markdown

```diff
@@ -9,5 +9,5 @@ We think it's a great project, with a great team, and we're proud to have been i
 [MOOC]: http://en.wikipedia.org/wiki/Massive_open_online_course
 
 :kind: project
-:display_name: "FutureLearn"
+:display_name: "FutureLearn MVP"
```

Comment:
I think the "MVP" bit of the project name came from the original contract. However, I think this contract was then just extended and the name of the project was not changed, even though we were then working way beyond an MVP. I think the MVP only covers the first few months at FL and I think it would be good to think of a better name for the project in here. However, I don't think it should stop you merging these changes.